cfgov/flags/templates/flagadmin/flags/create.html

```diff
@@ -0,0 +1,18 @@
+{% extends "wagtailadmin/base.html" %}
+{% load i18n %}
+{% load gravatar %}
+{% block titletag %}{% trans "Flags" %}{% endblock %}
+
+{% block content %}
+    {% trans "Flags" as flags_str %}
+    {% trans "Add a flag" as add_a_flag_str %}
+
+    {% include "wagtailadmin/shared/header.html" with title=flags_str   %}
+    
```

Comment:
ooo so we'll be able to create flags in wagtail? how would this work? 
i'm assuming whole navigation flows would be feature flipped off, but what would be the diff between then being unpublished.
or maybe i'm just thinking about this incorrectly